flow/include/flow/EncryptUtils.h

```diff
@@ -37,14 +37,19 @@
 
 #define SYSTEM_KEYSPACE_ENCRYPT_DOMAIN_ID -2
 #define ENCRYPT_HEADER_DOMAIN_ID -3
-
-const std::string FDB_DEFAULT_ENCRYPT_DOMAIN_NAME = "FdbDefaultEncryptDomain";
+#define FDB_DEFAULT_ENCRYPT_DOMAIN_ID -4
 
 using EncryptCipherDomainId = int64_t;
 using EncryptCipherDomainNameRef = StringRef;
+using EncryptCipherDomainName = Standalone<EncryptCipherDomainNameRef>;
 using EncryptCipherBaseKeyId = uint64_t;
 using EncryptCipherRandomSalt = uint64_t;
 
+const EncryptCipherDomainNameRef FDB_DEFAULT_ENCRYPT_DOMAIN_NAME_REF =
+    EncryptCipherDomainNameRef(std::string("FdbDefaultEncryptDomain"));
+const EncryptCipherDomainName FDB_DEFAULT_ENCRYPT_DOMAIN_NAME =
+    EncryptCipherDomainName(std::string("FdbDefaultEncryptDomain"));
```

Comment:
Do you need to have a ref and non-ref version of this? Also, you shouldn't need to convert from a string, instead do:


```
const EncryptCipherDomainNameRef FDB_DEFAULT_ENCRYPT_DOMAIN_NAME = "FdbDefaultEncryptDomain"_sr;

Reply:
Seems like a fallout from my patch, `FDB_DEFAULT_ENCRYPT_DOMAIN_NAME` is not used and can be removed. 

Will the transform the code as per suggestion, thanks.